publisher/src/utils/helperUtils.ts

```diff
@@ -276,3 +277,20 @@ type FormatSystemNameOptions = {
   allUserSystems?: AgencySystems[];
   hideCombined?: boolean;
 };
+
+// Only display one "JC Admin" name if there are multiple "JC Admin" editors
+export const filterJCAdminEditors = (editors: ReportEditor[]) => {
+  const result: ReportEditor[] = [];
+  let hasSeenJCAdmin = false;
+  editors.forEach((editor) => {
+    if (editor.name === "JC Admin") {
+      if (!hasSeenJCAdmin) {
+        hasSeenJCAdmin = true;
+        result.push(editor);
+      }
+    } else {
+      result.push(editor);
+    }
```

Comment:
nit: could this be simplified to the below?

```suggestion
    if (editor.name === "JC Admin" && !hasSeenJCAdmin) {
        hasSeenJCAdmin = true;
    }
    result.push(editor);
```